### PR TITLE
Misc updating and refreshing

### PR DESCRIPTION
**Pull request by rudolphpienaar**
Update verbosity handling and add in-line help. No core logic changes.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ####################
-pfioh - v1.5.14.2
+pfioh - v1.5.16.0
 ####################
 
 .. image:: https://badge.fury.io/py/pman.svg
-    :target: https://badge.fury.io/py/pman
+    :target: https://badge.fury.io/py/pfioh
```
Comment (danmcp) [inline README.rst]: Not sure I understand this one

**Comment by danmcp** [comment]
@rudolphpienaar I had one question and it would be good to squash the commits.

File: README.rst (hunk 1 above)
Comment (rudolphpienaar) [reply README.rst]: So once upon a time I copy pasted from ``pman`` when I created the repo, and those references should have been pfioh for this repo. Does that make sense? Not sure what exactly you are asking about...

Comment (danmcp) [reply README.rst]: Yes, but is line 5 still correct then?

Comment (rudolphpienaar) [reply README.rst]: great catch!

**Pull request merged**